tests/integration/test_data/gomod_e2e_test/container/Containerfile

```diff
@@ -0,0 +1,16 @@
+FROM docker.io/golang:1.18.1 AS build
+
+# Print cachi2 env vars file
+RUN cat /tmp/cachi2.env
+
+# Check content of source repository folder
+RUN ls /tmp/gomod_e2e_test-source/
+
+WORKDIR /tmp/gomod_e2e_test-source
+RUN . /tmp/cachi2.env && go build -o /retrodep
+
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.0.0
+
+COPY --from=build /retrodep /usr/bin/retrodep
```

Comment:
IMO we don't need a second stage, single stage should be fine to enough that the build works (and should be faster since we won't need to pull a second base image)

Reply:
Ok changed to single stage build